examples/sam2_amg_server/README.md

```diff
@@ -41,7 +41,26 @@ The 'ao' mode is a copy of the baseline with modifications to make the code more
 ### 0. Download checkpoints and install requirements
 
 ```
-pip install -r requirements.txt
+# From the top-level "ao" directory
+
+# If necessary, create and activate a virtual environment
+# Ex:
+python -m venv venv && source venv/bin/activate
+
+# Install requirements for this example
+pip install -r examples/sam2_amg_server/requirements.txt
+
+# If you have an older version of torch in your current environment, uninstall it first
+pip uninstall torch
+
+# Install torch nightly
+pip install torch torchvision torchaudio
```

Comment:
Hm, I think this would download the stable versions. You probably need to use the `--pre` variants again unless there's something else that causes nightlies to be selected.